### DISCUSSION TITLE
How I can donate or support development?

### DISCUSSION THREAD

**Discussion by xcv1001** [YOURLS/YOURLS#3241] [Q&A]
HI

FIRST OF ALL, THANK YOU FOR SAVING US 1000000s USD. 
We are a small SME and do not have a big budget to buy services from bit,ly and other companies.

Recently BIT.LY stopped giving free branded short URLs and we were stuck with a nightmare!!!
Thank you so much for your awesome wicked plugin we can now host on our own server and keep it under control!!

I would like to contribute one time please tell me how?

If you have Paypal please share me your address

**Comment by ozh** [comment]
Glad you enjoyed YOURLS :)

All the info is here: https://github.com/YOURLS/YOURLS/wiki/Donations
